src/__tests__/management.spec.ts

```diff
@@ -664,4 +664,40 @@ describe("Management", (): void => {
             expect(response).toBeTruthy();
         });
     });
+
+    describe("AllowedOriginsMerchantLevelApi", (): void => {
+        test("Delete an allowed origin", async () => {
+            scope.delete("/merchants/foo/apiCredentials/BAR123/allowedOrigins/fishy%20one").reply(204);
+
+            await management.AllowedOriginsMerchantLevelApi
+                .deleteMerchantsMerchantIdApiCredentialsApiCredentialIdAllowedOriginsOriginId("foo", "BAR123", "fishy one");
+        });
+
+        test("Create an allowed origin", async () => {
+            const requestBody = {
+                "domain": "https://www.eu.mystore.com"
+            };
+            scope.post("/merchants/YOUR_MERCHANT_ACCOUNT/apiCredentials/YOUR_API_CREDENTIAL/allowedOrigins", requestBody)
+                .reply(200, {
+                    "id": "YOUR_ALLOWED_ORIGIN",
+                    "data": [
+                        {
+                            "domain": "https://www.eu.mystore.com",
+                        }
+                    ], 
+                    "_links": {
+                        "self": {
+                            "href": "https://management-test.adyen.com/v1/merchants/YOUR_MERCHANT_ACCOUNT/apiCredentials/YOUR_API_CREDENTIAL/allowedOrigins/YOUR_ALLOWED_ORIGIN"
+                        }
+                    }
+                });
+
+            const response: AllowedOriginsResponse = await management.AllowedOriginsMerchantLevelApi
+                .postMerchantsMerchantIdApiCredentialsApiCredentialIdAllowedOrigins("YOUR_MERCHANT_ACCOUNT", "YOUR_API_CREDENTIAL", {
```

Comment:
This is an `operationId` based name,  the actual name will be come later. The rest of the test/code shouldn't change.